schemas/bursar-export/bursarExportJob.json

```diff
@@ -0,0 +1,52 @@
+{
+  "description": "Bursar export job schema",
+  "type": "object",
+  "properties": {
+    "filter": {
+      "description": "Filter to use for the regular export",
+      "$ref": "filter/bursarExportFilter.json"
+    },
+    "groupByPatron": {
+      "description": "Choose whether to group fee/fines by patron",
+      "type": "boolean"
+    },
+    "groupByPatronFilter": {
+      "description": "Filter to use for the export when grouping by patron",
+      "$ref": "filter/bursarExportFilterAggregate.json"
+    },
+    "header": {
```

Comment:
it's not self-descriptive, our entity is Job (in this case BursarExport), what is the header in context of job

why not to call it exportHeaderFormat? and in description to add more info what formats, tokens etc can be used there

as for me, current description just explains name

Reply:
What would you propose instead? I'm not sure what name would be any clearer...

Reply:
I already did a proposal in the first comment + in another [thread](https://github.com/folio-org/folio-export-common/pull/64#discussion_r1478325553), just waiting a confirmation that my understanding is correct